# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLMethodDispatcher.h
# Commit: adc841b3e204
# Full Hash: adc841b3e204564ee8515a87206eb317275844c4
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-02-21 09:51:10
# Regressor Bug: 1616715
# File Overlap Count: 1
# Description:
#   Bug 1616715 - SurfaceFromElement gets alpha Premult unless opt-in to NonPremult. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D63421
# ==============================================================================

diff -r 7917b3a4a9a7 -r adc841b3e204 dom/canvas/WebGLMethodDispatcher.h
--- a/dom/canvas/WebGLMethodDispatcher.h	Fri Feb 21 03:11:23 2020 +0000
+++ b/dom/canvas/WebGLMethodDispatcher.h	Fri Feb 21 02:27:08 2020 +0000
@@ -52,7 +52,6 @@
 DEFINE_ASYNC(HostWebGLContext::DeleteVertexArray)
 
 DEFINE_ASYNC(HostWebGLContext::ClearVRFrame)
-DEFINE_SYNC(HostWebGLContext::GetSurfaceSnapshot)
 DEFINE_SYNC(HostWebGLContext::GetVRFrame)
 
 DEFINE_ASYNC(HostWebGLContext::Disable)